### PR TITLE
DOC: Minor documentation improvements 

### PR DESCRIPTION
**Pull request by nikitajz**

### CODE REVIEW

File: numpy/doc/creation.py [hunk 1]
```diff
@@ -9,8 +9,7 @@
 There are 5 general mechanisms for creating arrays:
 
 1) Conversion from other Python structures (e.g., lists, tuples)
-2) Intrinsic numpy array array creation objects (e.g., arange, ones, zeros,
-   etc.)
+2) Intrinsic numpy array creation objects (e.g., arange, ones, zeros, etc.)
```
Comment (mattip) [inline numpy/doc/creation.py]: now the line is 81 chars, which is over the limit (80)

File: numpy/doc/creation.py [hunk 2]
```diff
@@ -44,10 +43,10 @@
 NumPy has built-in functions for creating arrays from scratch:
 
 zeros(shape) will create an array filled with 0 values with the specified
-shape. The default dtype is float64.
+shape. The default dtype is float64. ::
 
-``>>> np.zeros((2, 3))
-array([[ 0., 0., 0.], [ 0., 0., 0.]])``
+ >>> np.zeros((2, 3))
+ array([[ 0., 0., 0.], [ 0., 0., 0.]])
```
Comment (mattip) [inline numpy/doc/creation.py]: Indeed, the page is currently rendered improperly http://www.numpy.org/devdocs/user/basics.creation.html#intrinsic-numpy-array-creation

**Comment by eric-wieser** [comment]
Is numpy/doc/*.py rendered at all?

**Comment by mattip** [comment]
Yes, as <root>/user/basics.creation.html

File: numpy/doc/creation.py (hunk 2 above)
Comment (eric-wieser) [reply numpy/doc/creation.py]: I think normally we'd indent this four spaces, not one

**Comment by charris** [comment]
Note to reviewers: If the fixes are minor -- spelling, long lines, etc. -- it is OK to edit the source to fix them.

File: numpy/doc/creation.py (hunk 2 above)
Comment (eric-wieser) [reply numpy/doc/creation.py]: Although what you have is consistent with the rest of the file, so nevermind

File: numpy/doc/creation.py (hunk 1 above)
Comment (eric-wieser) [reply numpy/doc/creation.py]: Line length didn't change here

**Review by eric-wieser — APPROVED** [review]
Looks good to squash and merge with a DOC: commit message - I'll try to remember to come back to this once tests pass

**Comment by charris** [comment]
So much for the github change merge base unfunctionality. What a disaster :() I'll fix this up.

**Comment by eric-wieser** [comment]
All it does is make `git merge <the-merge-base>` be what the merge button does - it sounds like you were looking for a rebase button

**Comment by charris** [comment]
Fixed up in #11287. Not sure if there was a way to do it directly on github, but redoing the PR looked like the ease way.

@eric-wieser Ping.

**Comment by charris** [comment]
@nikitajz Sorry about that.

**Comment by charris** [comment]
@eric-wieser I tried going back and forth on the rebases to see if it would fix itself up. I was hoping it would be a simple rebase of the commits in the PR, but no dice.

**Comment by nikitajz** [comment]
@charris thanks. Sorry if it was too much hassle for you.

**Pull request closed without merging**